package.json

```diff
@@ -69,15 +69,16 @@
   },
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
+    "@ember/test-helpers": "^2.8.1",
     "@formatjs/intl-pluralrules": "^4.0.6",
     "@formatjs/intl-relativetimeformat": "^8.0.4",
     "@glimmer/component": "^1.0.3",
     "@glimmer/tracking": "^1.0.3",
     "@types/ember": "^3.16.3",
-    "@types/ember-qunit": "^3.4.13",
```

Comment:
[`ember-cli-qunit`](https://github.com/ember-cli/ember-cli-qunit) has been deprecated. I replaced it with [`ember-qunit`](https://github.com/emberjs/ember-qunit). `v5` requires that we also install `@ember/test-helpers` and `qunit`.